packages/ui/src/components/ToggleSection/ToggleSection.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import type { ComponentStory, ComponentMeta } from "@storybook/react";
+import { AcademicCapIcon } from "@heroicons/react/24/solid";
+
+import { action } from "@storybook/addon-actions";
+import ToggleSection from "./ToggleSection";
+import Button from "../Button";
+
+const Toggle = () => {
+  const context = ToggleSection.useContext();
+
+  return (
+    <ToggleSection.Trigger>
+      <Button mode="inline">{context?.open ? "Close" : "Open"} Section</Button>
+    </ToggleSection.Trigger>
+  );
+};
+
+export default {
+  component: ToggleSection.Root,
+  title: "Components/Toggle Section",
+} as ComponentMeta<typeof ToggleSection.Root>;
+
+const Template: ComponentStory<typeof ToggleSection.Root> = (args) => (
+  <ToggleSection.Root
+    {...args}
+    onOpenChange={(open) => action("onOpenToggle")(open)}
+  >
+    <ToggleSection.Header Icon={AcademicCapIcon} toggle={<Toggle />}>
+      Toggle Section
+    </ToggleSection.Header>
+
+    <ToggleSection.Content data-h2-text-align="base(center)">
+      <ToggleSection.InitialContent>
+        <p>Initial Content Here</p>
+        <ToggleSection.Open>
+          <Button mode="inline">Open</Button>
```

Comment:
Is there a way to test this AC?
> focus goes to first editable input on the edit mode

It seems like currently there's no focus change on toggle.

Reply:
Oh yeah, missed that note, thanks.